sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.integration.service.ts

```diff
@@ -55,7 +59,9 @@ export class IER12IntegrationService extends SFTPIntegrationBase<void> {
         ierRecord.studentMaritalStatusCode;
       ierFileDetail.addressInfo = ierRecord.addressInfo;
       ierFileDetail.programName = ierRecord.programName;
-      ierFileDetail.programDescription = ierRecord.programDescription;
+      ierFileDetail.programDescription = replaceLineBreaks(
```

Comment:
just wanted know, Is this the only field in IER, where there can be a possible line break? or are we only considering `programDescription` as part of the PR?

Reply:
As discussed, programDescription  was the only text area input which is part of IER12 data.

Reply:
Thanks for confirming. resolving